monai/transforms/spatial/array.py

```diff
@@ -1372,6 +1399,9 @@ def __call__(
 class RandAffine(RandomizableTransform):
     """
     Random affine transform.
+    More details about affine matrix definition: https://www.slicer.org/wiki/Coordinate_systems.
```

Comment:
There's no need to mention the coordinate system for these data augmentation classes, the affine transforms are not specific to it

Reply:
updated in the latest commit.
Thanks.